pkg/virt-operator/resource/generate/rbac/cluster.go

```diff
@@ -715,3 +720,33 @@ func newInstancetypeViewClusterRole() *rbacv1.ClusterRole {
 		},
 	}
 }
+
+func newInstancetypeViewClusterRoleBinding() *rbacv1.ClusterRoleBinding {
```

Comment:
Why don't you aggregate to the kubernetes view role like `kubevirt.io:view` does? This saves you from creating an additional binding.

Reply:
Because the aggregated `kubevirt.io:view` role is bound with a `RoleBinding`, which does not grant access to cluster wide resources.

Reply:
See https://github.com/kubevirt/kubevirt/issues/10437 for reference